bin/asl-setup-dkms-mok

```diff
@@ -0,0 +1,84 @@
+#!/usr/bin/bash
+#
+
+if [ ${UID} -ne 0 ]; then
+	echo "Must be run as root"
+	exit 1
+fi
+
+if [ ! -x /usr/bin/mokutil ]; then
+	echo "Missing /usr/bin/mokutil"
+	exit 1
+fi
+
+SBSTATE=$(mokutil --sb-state | awk '{print $2}')
+if [ "${SBSTATE}" != "enabled" ]; then
```

Comment:
Let's say I have a system with Secure Boot "disabled".
Let's also say that I "want" to enable Secure Boot.
Ignoring this check, could I run the script to prepare my system for Secure Boot, then enable Secure Boot during the restart process, and then when booting into the now "enabled" system all would be good?

Reply:
SecureBoot isn't really something you can just "turn on".